tests/integration.rs

```diff
@@ -203,3 +205,21 @@ fn tool_test_ingredient_folder() -> Result<(), Box<dyn std::error::Error>> {
     assert!(json.contains("manifest_data"));
     Ok(())
 }
+
+#[test]
+// c2patool tests/fixtures/earth_apollo17.jpg -m tests/fixtures/ingredient_test.json -fo target/tmp/ingredients.jpg
```

Comment:
I like the idea of including the command as a comment above the test for `Command::cargo_bin` like tests 👍